plugin/docker/container-log.go

```diff
@@ -18,7 +18,7 @@ type containerLogFile struct {
 	client        *client.Client
 }
 
-func containerLogFileBase() *containerLogFile {
+func containerLogFileBase(forInstance bool) *containerLogFile {
```

Comment:
There is one subtlety here. If an entry's attributes are set but `meta` is not provided, `meta` defaults to returning a hash of those attributes (to be consistent with the "attributes are a subset of the metadata" idea). The current code does not work with those entries because there isn't any attributes schema yet so I can't construct a suitable default `meta` schema. Guess that could be a TODO or something to look at next, although I don't think it will be as valuable since lots of things support `ctime`, `mtime`, etc.

Reply:
`containerLogFileBase` is a convention, not an interface you have to conform to. If you're not going to use `forInstance` in this case, why add it?